stock_picking_validate/wizard/stock_picking_validate_wizard.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="stock_picking_validate_wizard" model="ir.ui.view">
+        <field name="name">stock.picking.validate.wizard</field>
+        <field name="model">stock.picking.validate</field>
+        <field name="arch" type="xml">
+            <form string="Stock Picking Validate">
+                <p class="oe_gray">This wizard will allow validate bunch of Stock Picking.</p>
```

Comment:
"This wizard will validate selected stock pickings at once. Make sure that you have made no mistake with selection."